airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeInsertDestination.java

```diff
@@ -26,7 +27,7 @@ public SnowflakeInsertDestination() {
 
   @Override
   protected DataSource getDataSource(final JsonNode config) {
-    return SnowflakeDatabase.createDataSource(config);
+    return SnowflakeDatabase.createDataSource(config, OssCloudEnvVarConsts.AIRBYTE_OSS);
```

Comment:
is this class even still in use? I.e. can we just delete this file completely

Reply:
Nice catch. I haven't checked carefully yet, it was automatically updated by refactor IntelliIdea's feature. At first glance it's not used at all. Removed for now, will check later if it cause any issues